### PR TITLE
[regression-test](array) fix abnormal test on function array_intersect

### PR DESCRIPTION
**Pull request by eldenmoon**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yiguolei — DISMISSED** [review]
LGTM

File: regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions_by_literal.groovy [hunk 1]
```diff
@@ -190,6 +190,6 @@ suite("test_array_functions_by_literal") {
     // abnormal test
     test {
         sql "select array_intersect([1, 2, 3, 1, 2, 3], '1[3, 2, 5]')"
-        exception "errCode = 2, detailMessage = No matching function with signature: array_intersect(array<tinyint(4)>, varchar(-1))"
+        exception "errCode = 2, detailMessage = No matching function with signature: array_intersect(array<tinyint(4)>, varchar(65533))"
```
Comment (cambyzju) [inline regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions_by_literal.groovy:193]: For this case, we maybe only need to catch error msg prefix: `errCode = 2, detailMessage = No matching function with signature: array_intersect`

Comment (eldenmoon) [reply regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions_by_literal.groovy:193]: you're right, but the pipeline is blocked， It's just a quick fix ，I will fix it in latter PR

Comment (eldenmoon) [reply regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions_by_literal.groovy]: fixed

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**